{
  final Exception e=new Exception(EXCEPTION_MSG,new Exception(EXCEPTION_CAUSE));
  throw new YarnException(e);
}
